res/controllers/common-controller-scripts.js

```diff
@@ -368,6 +368,7 @@ bpm.tapButton = function(deck) {
 // ----------------- Common regular expressions --------------------------
 script.samplerRegEx = /\[Sampler(\d+)\]/ ;
 script.channelRegEx = /\[Channel(\d+)\]/ ;
+script.channelRegExExact = /^\[Channel(\d+)\]$/ ;
```

Comment:
> I mean there may be a use case for pulling [Channel1] out of [EqualizerRack1_[Channel1]_Effect1]. eqRegEx and quickEffectRegEx would not do that.

The inner group is a capture -- so you can:
```javascript
"[EqualizerRack1_[Channel1]_Effect1]".match(eqRegEx)
```
Returns:
```javascript
["[EqualizerRack1_[Channel1]_Effect1]", "Channel1"]
```

> I don't understand what you mean.

See in the above example how the capture is "Channel1" -- you would have to append square brackets on either side to get the group. I'm suggesting either removing the square brackets from either side of the capture or sticking them inside the parentheses so you don't have to append square brackets after matching. 

If that takes care of the use case of extracting "inner" groups then I'd suggest changing all the regexes to have ^$ instead of adding "exact" variants of each -- generally I feel fewer exposed symbols from our common libs is better.